### ISSUE TITLE
Missing Utc module "FILE NOT FOUND"

### ISSUE THREAD

**Issue by tannerntannern** [roc-lang/basic-cli#34]
Trying to access the `Utc` module that appears to be part of the basic-cli

```elm
app "myapp"
    packages { pf: "https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br" }
    imports [
        pf.Utc,
    ...
```

```
$ roc dev
Downloading https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br
    into <user>/.cache/roc/packages

── FILE NOT FOUND ──────────────────────────────────────────────── UNKNOWN.roc ─

I am looking for this file, but it's not there:

    <user>/.cache/roc/packages/github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4/Utc.roc

Is the file supposed to be there? Maybe there is a typo in the file
name?
```

When I inspect that directory, indeed there is no `Utc.roc` file.  It's not clear to me whether this is intended to be a public module, but the functions inside looked useful.

**Comment by lukewilliamboswell** [comment]
I suspect that it because it hasn't made it into the 0.3.1 release yet. If you clone basic-cli and use a rellative link you should be a le to use it as a work around for the package URL. 

I think the releases are still made manually, and Utc was only added recently, so this should be resolved on the next update.

**Comment by Anton-4** [comment]
I'll do a new release next week.

**Comment by tannerntannern** [comment]
Thanks -- I'll resolve the issue since it just hasn't been released

**Comment by Anton-4** [comment]
I've [released basic-cli 0.3.2](https://github.com/roc-lang/basic-cli/releases).
